numalogic/tools/trainer.py

```diff
@@ -80,5 +80,5 @@ def predict(self, model: Optional[LightningModule] = None, unbatch=True, **kwarg
         recon_err = super().predict(model, **kwargs)
         recon_err = torch.vstack(recon_err)
         if unbatch:
-            return inverse_window(recon_err, method="keep_last")
+            return inverse_window(recon_err, method="keep_first")
```

Comment:
any specific reason for this?

Reply:
Better reconstruction if we `keep_first`